packages/Webkul/Admin/src/Resources/views/account/edit.blade.php

```diff
@@ -94,19 +89,71 @@ class="primary-button"
 
                         <x-admin::form.control-group.error control-name="email" />
                     </x-admin::form.control-group>
+
+                    <x-admin::form.control-group class="mb-4">
+                        <x-admin::form.control-group.label class="required">
+                            @lang('admin::app.account.edit.ui-locale')
+                        </x-admin::form.control-group.label>
+
+                        @php
+                        $locales = core()->getAllActiveLocales();
+                        @endphp
+
+                        <x-admin::form.control-group.control
+                            type="select"
+                            id="ui_locale_id"
+                            name="ui_locale_id"
+                            rules="required"
+                            :value="old('ui_locale_id') ?: $user->ui_locale_id"
+                            :label="trans('admin::app.account.edit.ui-locale')"
+                            :placeholder="trans('admin::app.account.edit.ui-locale')"
+                            :options="$locales"
+                            track-by="id"
+                            label-by="name"
+                        >
+                        </x-admin::form.control-group.control>
+
+                        <x-admin::form.control-group.error control-name="ui_locale_id" />
+                    </x-admin::form.control-group>
+
+                    <!-- TImezone -->
+                    <x-admin::form.control-group class="!mb-0">
+                        <x-admin::form.control-group.label class="required">
+                            @lang('admin::app.account.edit.user-timezone')
+                        </x-admin::form.control-group.label>
+
+                        @php
+                        $timezones = json_encode(core()->getTimeZones());
+                        @endphp
+
+                        <x-admin::form.control-group.control
+                            type="select"
+                            name="timezone"
+                            rules="required"
+                            :value="old('timezone') ?: $user->timezone"
+                            :label="trans('admin::app.account.edit.user-timezone')"
+                            :placeholder="trans('admin::app.account.edit.user-timezone')"
+                            :options="$timezones"
```

Comment:
Here too the options can be added like this without the need of an extra variable:
```php
:options="json_encode(core()->getTimeZones())"
```